src/components/Onfido/BaseOnfidoWeb.js

```diff
@@ -104,7 +104,12 @@ class Onfido extends React.Component {
             onModalRequestClose: () => {
                 Log.hmmm('Onfido user closed the modal');
             },
-            language: this.props.preferredLocale,
+            language: {
+                locale: this.props.preferredLocale,
+                phrases: {
+                    'generic.back': this.props.translate('common.back'),
+                },
+            },
```

Comment:
Let's add the Onfido custom localized doc here to clarify where this comes from.

Reply:
I would say something like this.
```javascript
// Provide a custom phrase for the back button so that the first letter is capitalized, and translate the phrase while we're at it. See the issue and documentation for more context.
// https://github.com/Expensify/App/issues/17244
// https://documentation.onfido.com/sdk/web/#custom-languages
```